src/framework/preferences/js/Panels.js

```diff
@@ -662,13 +673,25 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         },
         selectors: {
             textSize: ".flc-prefsEditor-min-text-size",
-            label: ".flc-prefsEditor-min-text-size-label",
+            label: ".flc-prefsEditor-min-text-size-label"
+        },
+        selectorsToIgnore: ["textSize"],
+        protoTree: {
+            label: {messagekey: "textSizeLabel"},
+            textSizeDescr: {messagekey: "textSizeDescr"},
+            smallIcon: {messagekey: "textSizeSmallIcon"},
+            largeIcon: {messagekey: "textSizeLargeIcon"},
+            multiplier: {messagekey: "multiplier"}
+        }
+    });
+
+    fluid.defaults("fluid.prefs.textSize.textfieldSlider", {
```

Comment:
I tried to do that, but  the options did not get pass on as they were supposed to. I will text you tomorrow to explain the problem in more details.